### ISSUE TITLE
Possible new release?

### ISSUE THREAD

**Issue by GitMensch** [geohot/qira#264]
Currently there are 79 commits since the last release (from 2019), so it seems useful to release. Are there any specific reasons to favor 1.3 over current master?

**Comment by elazarl** [comment]
@GitMensch  There seem to be other projects trying to tackle the same problem. Notable [rr](https://rr-project.org/) in the open source, and [udb](https://undo.io/products/udb) in the propriety area.

**Comment by GitMensch** [comment]
Yes, these projects tackle "in general" the same problem, but they have different dependencies (for example RR won't work with an older AMD) and there are still [those 79 commits since last release](https://github.com/geohot/qira/compare/v1.3...master). Therefore I'd still suggest to have "at least a 'final' 1.4 release of qira.

Furthermore there's the open question "should one favor 1.3 over current master - if yes, why?".
